users/views.py

```diff
@@ -1,3 +1,47 @@
+import json, re, bcrypt
 
+from django.http import JsonResponse
+from django.views import View
+from django.core.exceptions import ValidationError
+
+
+from users.validation import email_check, password_check
+from .models import User
+
+# Create your views here.
+
```

Comment:
import convention에 맞춰주세요.
```suggestion
import json
import re 

import bcrypt
from django.http            import JsonResponse
from django.views           import View
from django.core.exceptions import ValidationError

from .validation import email_check, password_check
from .models     import User

```

Reply:
수정하겠습니다!

Reply:
수정하겠습니다!